tests/zfs-tests/include/libtest.shlib

```diff
@@ -3917,8 +3933,18 @@ function get_tunable_impl
 	FreeBSD)
 		sysctl -n vfs.zfs.$tunable
 		;;
-	SunOS)
+	dilos)
 		[[ "$module" -eq "zfs" ]] || return 1
+		typeset -li value=0
+		# value=$(mdb -k -e "$tunable/X | ::eval .=U" | sed -e 's/^[[:space:]]*//')
```

Comment:
Let's remove the commented lines entirely (here and below) if they're not correct for the platform.